problems/arrayMethods.js

```diff
@@ -45,7 +71,12 @@ const stringsOnly = () => {};
  * @returns {number[]} Only even valued elements
  */
 
-const onlyEvens = () => {};
+const onlyEvens = (nums) => {
+  return nums.filter((el) => {
+    return el %2 === 0
```

Comment:
Nice, don't forget to format your code correctly like a code formatter like prettier ( alt + shift + f )